app/views/projects/blob/_editor.html.haml

```diff
@@ -17,7 +17,10 @@
 
   .file-content.code
     %pre.js-edit-mode-pane#editor
-      = params[:content] || local_assigns[:blob_data]
+      - content = params[:content] || local_assigns[:blob_data]
+      = content
+      - if content.length && 0 === content[content.length-1].index(/[\r\n]/)
```

Comment:
I'm a bit confused how this works. There's an `if` clause here with a non-printing line.